Sources/Networking/ETagManager.swift

```diff
@@ -130,6 +124,8 @@ private extension ETagManager {
         return request.url?.absoluteString
     }
 
+    // TODO: delete old data since this isn't backwards compatible
```

Comment:
Instead of deleting the old data, I added a test in #1438 to ensure that `ETagManager` fails gracefully and treats it as a cache miss. How does that sound?